### PR TITLE
Adds a new slime type

### PR DESCRIPTION
**Pull request by Mlgdatboiwew**
This pull if merged will add a new slime that is obtainable via xenobiology, staff of change or admins after receiving a wizard slime you will have to find a chemical called strange reagent which is made from holy water, synth flesh and a few other things, After injecting the slime core with Strange Reagent you will temporary blinded with a flash light and then a random spell book will appear at your feet.The science/reasoning behind this is that the wizard federation was finding a cheaper way to produce things like wands and spell books because they were only one use so they decided to experiment with the slime species and after years of handwork a gold slime evolved into a wizard one but sadly a cent comm death squad raided the wizard federation science facility in resulted in the slime hiding in their ship and leaking into the cycle of slimes.Why did I write all of that?

+New Wizard slime

By the way hopefully the sprites are placeholder if anyone wants to help replace them message me on discord my name is Hi.

This addition could make end rounds funner for players instead of farming all round for a gold and pink slime they could just get a wizard slime this would help players not be as annoyed doing a ton of progress and losing it and can instead play around with a variety of spells from anything like Fireballs to smoke based off of how lucky you are!

### CODE REVIEW

**Comment by McBawbaggings** [comment]
You'll need to unselect the original TGstation versions of your files in hippiestation.dmb. That's why the code isn't compiling.
I think this could be modularised more nicely as well.

Changelog isn't quite working either.
```:cl: YourNameHere
add: Did some slime changes
/:cl:```

File: hippiestation/code/modules/mob/living/simple_animal/slime/slime.dm [hunk 1]
```diff
@@ -0,0 +1,421 @@
+/mob/living/simple_animal/slime
```
Comment (Pyko1) [inline hippiestation/code/modules/mob/living/simple_animal/slime/slime.dm]: Only mirror the changes.

ie.
```
/mob/living/simple_animal/slime
	var/static/list/slime_colours = list("rainbow", "grey", "purple", "metal", "orange",
 	"blue", "dark blue", "dark purple", "yellow", "silver", "pink", "red",
 	"gold", "green", "adamantine", "oil", "light pink", "bluespace",
 	"cerulean", "sepia", "black", "pyrite", "wizard")
```

Comment (Mlgdatboiwew) [reply hippiestation/code/modules/mob/living/simple_animal/slime/slime.dm]: wat

**Comment by Swagile** [comment]
i thought you were joking when you were talking about this in OOC, you absolute madman

this is great though, makes xenobio more than a "sentience potion hostile mobs when your bored" the simulator

File: hippiestation/code/modules/mob/living/simple_animal/slime/slime.dm (hunk 1 above)
Comment (Pyko1) [reply hippiestation/code/modules/mob/living/simple_animal/slime/slime.dm]: our module takes priority over the original one, so basically you just define what you are editing and then the changes and additions.

Comment (Pyko1) [reply hippiestation/code/modules/mob/living/simple_animal/slime/slime.dm]: hence you call what you edit aka
```
/mob/living/simple_animal/slime
```
and then only the parts you edit or add aka
```
	var/static/list/slime_colours = list("rainbow", "grey", "purple", "metal", "orange",
 	"blue", "dark blue", "dark purple", "yellow", "silver", "pink", "red",
 	"gold", "green", "adamantine", "oil", "light pink", "bluespace",
 	"cerulean", "sepia", "black", "pyrite", "wizard")
```
leaving out all the parts already handled by the normal .dm outside of hippie folder.

**Comment by PopNotes** [comment]
does he have any idea what he's doing

**Comment by PopNotes** [comment]
(no)

**Comment by Mlgdatboiwew** [comment]
I was never explained how to exactly do all this modular shit n stuff

**Comment by JamieH** [comment]
Closing this.

@Mlgdatboiwew read this: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md reopen when you're ready

**Pull request closed without merging**